src/components/genericTraderModal.html

```diff
@@ -0,0 +1,100 @@
+<div class="modal fade noselect" id="genericTraderModal" tabindex="-1" role="dialog"
+     aria-labelledby="genericTraderModalLabel">
+    <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header text-center">
+                <h4 data-bind='text: ShopHandler.shopObservable?.().name'></h4>
+                <button type="button" class="btn btn-primary" data-dismiss="modal">&times;</button>
+            </div>
+            <div>
+                <ul class="nav nav-tabs nav-fill">
+                    <li class="nav-item"><a data-toggle='tab' class='nav-link active' href="#genericTraderTrade">Trades</a></li>
+                </ul>
+            </div>
+            <div class="modal-body tab-content">
+                <div id="genericTraderTrade" class="tab-pane fade in active show" style="min-height: 250px;">
+                    <div style="display: grid; column-gap: 0.5rem; grid-template-columns: auto auto 1fr auto auto auto 1fr auto;">
+                        <div class="font-weight-bold" style="grid-column: 1 / 3">Cost</div>
+                        <div class="font-weight-bold">Inv</div>
+                        <div class="font-weight-bold"></div>
+                        <div class="font-weight-bold" style="grid-column: 5 / 7">Profit</div>
+                        <div class="font-weight-bold">Inv</div>
+                        <div class="font-weight-bold"></div>
+
+                        <!-- ko foreach: GenericDeal.getDeals(ShopHandler.shopObservable().traderID) -->
+                        <!-- ko let: { deal: $data } -->
```

Comment:
Can this `foreach` and `let` be simplified with this? https://knockoutjs.com/documentation/foreach-binding.html#note-3-using-as-to-give-an-alias-to-foreach-items

Reply:
Resolved